guides/src/content/user/configuration/configuring_analytics.md

```diff
@@ -6,6 +6,20 @@ title: Analytics Tracker
 
 Understanding your site's visitor traffic patterns are important for planning your company's marketing and growth strategies.
 
-### Google Analytics
+### Google Analytics Enhanced Ecommerce and Segment Ecommerce
 
-Spree's Admin Interface makes it easy for you to add the robust Google Analytics toolset to your site. [This blog entry](http://spreecommerce.org/pages/blog/ecommerce-tracking-in-spree) covers all of the intricacies with registering a Google Analytics account, adding the tracker to your site, and testing the functionality.
+Spree's Admin Interface makes it easy for you to add the robust Google Analytics and Segment toolset to your site.
```

Comment:
@wojtekwisniewski we need to add a link to https://github.com/spree-contrib/spree_analytics_trackers extension here as this was extracted from core Spree